### PR TITLE
DSE/new button layout prod 13226

### PR DESCRIPTION
**Pull request by sellisd**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: cypress/e2e/brewery/DatasetManager.cy.js [hunk 1]
```diff
@@ -333,8 +333,6 @@ describe('Refresh dataset', () => {
         finalIngestionStatus: 'ERROR',
       };
       DatasetManager.switchToDatasetManagerView();
-      DatasetManager.getDatasetRefreshButton(DATASETS_TO_REFRESH[0].id).should('be.visible');
-      DatasetManager.getDatasetRefreshButton(DATASETS_TO_REFRESH[1].id).should('be.visible');
```
Comment (csm-thu) [inline cypress/e2e/brewery/DatasetManager.cy.js:337]: Can we add some actions in the test to balance the removal of these lines?
For example, we could check that when we select a dataset in the list, the refresh button for this dataset is shown in the overview

File: src/views/DatasetManager/components/DatasetOverview/components/DatasetActions/DatasetActions.js [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+      </IconButton>
+      <IconButton>
+        <FileCopy color="primary" />
+      </IconButton>
+      <IconButton>
+        <AddCircleIcon color="primary" />
+      </IconButton>
+      <IconButton>
+        <ShareIcon color="primary" />
+      </IconButton>
```
Comment (csm-thu) [inline src/views/DatasetManager/components/DatasetOverview/components/DatasetActions/DatasetActions.js:25]: As discussed, we probably don't want to have fake buttons in our _main_ branch :smile:

**Review by csm-thu — CHANGES_REQUESTED** [review]
Suggested some changes :slightly_smiling_face:

File: cypress/e2e/brewery/DatasetManager.cy.js (hunk 1 above)
Comment (sellisd) [reply cypress/e2e/brewery/DatasetManager.cy.js:337]: I think this is tested a bit further down where I added line 347 to select the dataset before clicking on its refresh button

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**